packages/core/src/types.ts

```diff
@@ -2171,13 +2170,9 @@ export interface ActorRef<
   /** @internal */
   _processingStatus: ProcessingStatus;
   src: string | AnyActorLogic;
-  on: <TType extends TEmitted['type']>(
-    type: TType,
-    handler: (emitted: TEmitted & { type: TType }) => void
-  ) => Subscription;
```

Comment:
this could easily be seen as a breaking change

Reply:
How? It still shows up on the Actor type, which is what is returned from createActor. It shouldn't have been on ActorRef, since that's supposed to be a minimal interface to reference the actor. 

I can add it back, though.

Reply:
It was on an `ActorRef` as that's what u get from `.children` etc. Without it you can't easily subscribe to events emitted within the same "actor tree"

Reply:
Added back